shared/featureconfig/flags.go

```diff
@@ -173,6 +173,8 @@ var devModeFlags = []cli.Flag{
 	forceMaxCoverAttestationAggregation,
 	newBeaconStateLocks,
 	batchBlockVerify,
+	enableFinalizedDepositsCache,
```

Comment:
I decided to include this flag as well. Are there any guidelines as to what flags should be enabled in dev mode?